shared.env

```diff
@@ -4,5 +4,4 @@ EVENTSTORE_INT_TCP_PORT=1112
 EVENTSTORE_HTTP_PORT=2113
 EVENTSTORE_TRUSTED_ROOT_CERTIFICATES_PATH=/etc/eventstore/certs/ca
 EVENTSTORE_DISCOVER_VIA_DNS=false
-EVENTSTORE_ENABLE_EXTERNAL_TCP=true
 EVENTSTORE_ENABLE_ATOM_PUB_OVER_HTTP=true
```

Comment:
we may have some adjustments to make in the dockerfile in this repository and also in train station

Reply:
What kind of adjustments are we talking about?

Reply:
removing any references to database settings that are being removed

Reply:
done